packages/migrations/src/20210923155000-add-first-explanation-to-equation.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * This file is part of Serlo.org.
+ *
+ * Copyright (c) 2013-2021 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2021 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/serlo.org for the canonical source repository
+ */
+import { createEdtrIoMigration, updatePlugins } from './utils'
+
+createEdtrIoMigration({
+  exports,
+  migrateState: updatePlugins({
+    equations: ({ state }) => {
+      if (typeof state !== 'object' || state === null)
```

Comment:
Do we need that check?

Reply:
We only need the check so that the typescript compiler is happy since state has the type `unknown`